src/pages/faq/comparison/sst.mdx

```diff
@@ -0,0 +1,74 @@
+export const description = 'Comparison to SST'
+
+# SST vs. Nitric
+
+Nitric is a framework designed to aid developers in building full cloud applications on the Cloud of their choice, including declaring their infrastructure and application code in one place.
+SST is a framework that makes it easy to build modern full-stack JavaScript/TypeScript applications on AWS.
+
+These are the main differences between the two:
+
+1. **Cloud Provider Support:** Nitric is provider-agnostic, capable of deploying to multiple cloud providers such as AWS, Google Cloud, and Azure. SST is primarily designed for AWS services.
+2. **Language Support:** Nitric provides multiple libraries for TypeScript/JavaScript, Python, Go, C# .NET and Java, allowing developers to choose their preferred language. Currently SST only supports TypeScript and JavaScript.
+3. **Infrastructure Provisioning:** Nitric uses Pulumi by default for provisioning cloud resources, but also allows the use of [custom providers](/reference/providers/custom/building-custom-provider). SST uses the AWS CDK, which uses AWS CloudFormation for provisioning.
+4. **Local Simulation:** Nitric provides a fully offline local cloud simulation tool, allowing you to test your applications locally before deploying them. This is a feature that SST does not offer, for local development SST deploys your application to the cloud and uses a proxy to sync with your local code.
+
+## Code Comparison
+
+To get a deeper understanding of the differences, let's see the same app built in both Nitric and SST. The SST App in this example is standalone without the frontend code.
+
+<CodeGroup compare>
+
+```javascript {{title: 'Nitric', label: 'hello.ts' }}
+import { api } from '@nitric/sdk'
+
+const helloApi = api('main')
+
+helloApi.get('/hello/:name', async (ctx) => {
+  const { name } = ctx.req.params
+
+  ctx.res.body = `Hello ${name}`
+})
+```
+
+```javascript {{title: 'SST', label: 'stacks/hello-stack.ts'}}
+import { StackContext, Api } from 'sst/constructs'
+
+export function API({ stack }: StackContext) {
+  const api = new Api(stack, 'api', {
+    routes: {
+      'GET /hello/{name}': 'packages/functions/src/hello.handler',
+    },
+  })
+
+  stack.addOutputs({
+    ApiEndpoint: api.url,
+  })
+}
+```
+
+```javascript {{title: 'SST', label: 'packages/functions/hello.ts'}}
+import { ApiHandler, usePathParam } from 'sst/node/api'
+
+export const handler = ApiHandler(async (evt) => {
+  const name = usePathParam('name')
+
+  return {
+    statusCode: 200,
+    body: `Hello ${name}`,
+  }
+})
+```
+
```

Comment:
Might be worth adding a brief comparison at the end, I've also been flipping to examples around to put the application code side by side, followed by the infrastructure.